spec/SessionTokenCache.spec.js

```diff
@@ -2,49 +2,75 @@ const SessionTokenCache = require('../lib/LiveQuery/SessionTokenCache').SessionT
 
 describe('SessionTokenCache', function() {
 
-  beforeEach(function(done) {
-    const Parse = require('parse/node');
+  describe('Valid session', function() {
```

Comment:
As mentioned before, can we please not change all tests this way? But add new tests as we go?